docs/outputs.rst

```diff
@@ -161,40 +198,105 @@ Each row of the file corresponds to one time point found in the
-:abbr:`ICA (independent components analysis)`-:abbr:`AROMA (Automatic Removal Of Motion Artifacts)`
-(if enabled) are indicated with ``aroma_motion_XX``.
+Confound regressors description
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Basic confouds
+==================
+
+- ``trans_x``, ``trans_y``, ``trans_z``, ``rot_x``, ``rot_y``, ``rot_z`` - the 6 rigid-body motion
+  parameters (3 translations and 3 rotation) estimated relative to a reference image;
+
+- ``csf`` - the average signal within anatomically-derived eroded :abbr:`CSF (cerebro-spinal fluid)` mask;
+
+- ``white_matter`` - the average signal within  the anatomically-derived eroded :abbr:`WM (white matter)` masks;
+
+- ``global_signal`` -  the average signal within the brain mask.
+
+Parameter expansion of basic confounds
+=====================
+Regressing out the standard six motion parameters may not be sufficient to remove all variance related to head motion
+from the fMRI signal.
+Thus, Friston et al. ([Friston1996]_) and Satterthwaite et al. ([Satterthwaite2013]_)
+proposed *24-motion-parameter* expansion, with a goal of removing from fMRI signal as much of the motion-related
+variance as possible.
+To make this technique more accessible, *fMRIPrep* automaticaly calculates motion parameter
+expansion ([Satterthwaite2013]_), providing timeseries corresponding to first *temporal derivatives* of six motion
+parameters, together with their *quadratic terms*, resulting in the total 24 head motion parameters
+(6 standard motion parameters + 6 temporal derivatives of six motion parameters + 12 quadratic terms of 6 motion
+parameters and their 6 temporal derivatives).
+Additionally, *fMRIPrep* returns temporal derivatives and quadratic terms for the ``csf``, ``white_matter``
+and ``global_signal`` to enable applying 36-parameter denoising strategy
+proposed by Satterthwaite et al. ([Satterthwaite2013]_).
+
+Derivatives and quadratic terms are stored under column names with suffixes: ``_derivative1`` and powers ``_power2``.
+These were calculated for head motion estimates (``trans_`` and ``rot_``) and compartment signals
+(``white_matter``, ``csf``, and ``global_signal``).
+
+
+Confounds for outlier detection
+======================================
+
+- ``framewise_displacement`` - is a quantification of the estimated bulk-head motion calculated using
+  formula proposed by Power et al. ([Power2012]_);
+- ``dvars`` - the derivative of RMS variance over voxels (or :abbr:`DVARS`)([Power2012]_)
+- ``std_dvars`` - standardized DVARS;
+- ``non_steady_state_outlier_XX`` - columns indicate non-steady state volumes with a single
+  ``1`` value and ``0`` elsewhere (*i.e.*, there is one ``non_steady_state_outlier_XX`` column per
+  outlier/volume).
+
+All these confounds can be used to detect potential outlier time points - frames with high motion or spikes.
+Detected outliers can be further removed from time-series using methods such as: volume *censoring* - entirely
```

Comment:
```suggestion
Detected outliers can be further removed from time series using methods such as: volume *censoring* - entirely
```
Both "time-series and "time series" are used elsewhere. I think "time series" is preferred by many. My suggestions are for "time series" but I think it should just be consistent.